### PR TITLE
create DES Units of Measure

### PR DESCRIPTION
**Pull request by broughd-qld**
New vocabs for the Units of Measure and Quantity Kinds used by the DES Catalgoue.

Based on QUDT vocabularies only, some minor tweaking to the items and creation of new concept schemes.

### CODE REVIEW

File: vocabularies/qesd-unitofmeasure.ttl [hunk 1]
```diff
@@ -0,0 +1,809 @@
...
+@prefix qkdv: <http://qudt.org/vocab/dimensionvector/> .
+@prefix sou: <http://qudt.org/vocab/sou/> .
+@prefix qudt: <http://qudt.org/schema/qudt/> .
+
+<https://linked.data.gov.au/def/qesd-uom> a owl:Ontology ,
+        skos:ConceptScheme ;
+    dcterms:created "2021-04-15"^^xsd:date ;
+    dcterms:creator <https://linked.data.gov.au/org/des> ;
+    dcterms:modified "2021-04-16"^^xsd:date ;
+    dcterms:publisher <https://linked.data.gov.au/org/des> ;
```
Comment (KellyVance) [inline vocabularies/qesd-unitofmeasure.ttl:18]: For now at least, while we continue to use the gsq VocPrez I think:

```suggestion
    dcterms:publisher <https://linked.data.gov.au/org/gsq> ;
```

File: vocabularies/qesd-unitofmeasure.ttl [hunk 2]
```diff
@@ -0,0 +1,809 @@
...
+  qudt:iec61360Code "0112/2///62720#UAA033" ;
+  qudt:informativeReference "http://en.wikipedia.org/wiki/Celsius?oldid=494152178"^^xsd:anyURI ;
+  qudt:latexDefinition "\\(\\,^{\\circ}{\\rm C}\\)"^^qudt:LatexString ;
+  qudt:omUnit <http://www.ontology-of-units-of-measure.org/resource/om-2/degreeCelsius> ;
+  qudt:ucumCode "Cel"^^qudt:UCUMcs ;
+  qudt:uneceCommonCode "CEL" ;
+  qudt:unitOfSystem sou:CGS ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
+  skos:prefLabel "Degree Celsius"@en ;
+  skos:altLabel "degree-centigrade" ;
```
Comment (KellyVance) [inline vocabularies/qesd-unitofmeasure.ttl:215]:
```suggestion
  skos:altLabel "degree-centigrade"@en ;
```

File: vocabularies/qesd-unitofmeasure.ttl [hunk 3]
```diff
@@ -0,0 +1,809 @@
...
+  qudt:informativeReference "http://en.wikipedia.org/wiki/Litre?oldid=494846400"^^xsd:anyURI ;
+  qudt:omUnit <http://www.ontology-of-units-of-measure.org/resource/om-2/litre> ;
+  qudt:symbol "L" ;
+  qudt:symbol "l" ;
+  qudt:ucumCode "L"^^qudt:UCUMcs ;
+  qudt:ucumCode "l"^^qudt:UCUMcs ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
+  rdfs:label "Liter"@en-us ;
+  skos:prefLabel "Litre"@en ;
+  skos:altLabel "litre" ;
```
Comment (KellyVance) [inline vocabularies/qesd-unitofmeasure.ttl:766]:
```suggestion
  skos:altLabel "litre"@en ;
```

**Review by KellyVance — CHANGES_REQUESTED** [review]
Minor edits. 
But I think structure and content is otherwise ok.

**Review by nicholascar — CHANGES_REQUESTED** [review]
If I run the "Queensland Environmental Science Data Profile of QUDT Units" vocab against the VocPub validator* I see lots of validation errors that are a result of the QUDT ontology constructs - the particular labels they are using, the class relations and so on - getting in the way of what VocPrez or other SKOS system needs. For example, the first error I see is:

```turtle
[
	a sh:ValidationResult ;
	sh:resultSeverity sh:Violation ;
	sh:sourceConstraintComponent sh:UniqueLangConstraintComponent ;
	sh:sourceShape _:n172 ;
	sh:focusNode <http://qudt.org/vocab/quantitykind/RelativeHumidity> ;
	sh:resultPath skos:prefLabel ;
	sh:resultMessage "Language "en" has been used by 2 values" ;
	sh:resultMessage "Language "{?lang}" used more than once" ;
] .
```

This says that `quantitykind:RelativeHumidity` has two `skos:prefLabel` values in English. When I look at the vocab, you don't see this directly but note that the files contains this:

```turtle
quantitykind:RelativeHumidity
    ...
    owl:sameAs quantitykind:RelativePartialPressure ;
    ...
.
```

This says that `quantitykind:RelativeHumidity` is the same thing as `quantitykind:RelativePartialPressure` and that the two objects should share all properties, including `skos:prefLabel` values, thus it has two. This is fine in an ontology like QUDT but not fine for SKOS!

The solution is to reduce down the vocab file you have to just SKOS properties and constructs, i.e. not all of those present in QUDT. There is another reason for this too: you actually just want your vocab to list all the UoM for your org's use but if people really want other, detailed, QUDT properties for UoM conversion like `qudt:hasDimensionVector` (used for UoM -> UoM conversion) they they should go to QUDT and use the ontology.

See the QUDT profile I made for Loop3D (see the links under "Community Resources" on the QUDT home page at http://qudt.org/). The "Loop3d Vocabulary of Units" given there has the original QUDT URIs used but only about this much info per vocab concept:

```turtle
unit:KiloGM-PER-CentiM3
    a skos:Concept ;
    skos:definition "Kilogram per cubic centimetre is an SI derived unit of density, defined by mass in kilograms divided by volume in cubic centimetres. The official SI symbolic abbreviation is \\(kg \\cdot cm^{-3}\\), or equivalently either \\(kg/cm^3\\)."@en ;
    rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
    skos:prefLabel "kilogram per cubic centimetre"@en ;
    skos:altLabel "kilogram per cubic centimeter"@en-us ;
    skos:topConceptOf <http://qudt.org/community/loop3d/voc> ;
.
```

So the original entry in QUDT for `unit:KiloGM-PER-CentiM3` has lots of other stuff like:

```turtle
unit:KiloGM-PER-CentiM3
    ...
    qudt:conversionMultiplier 1000000.0 ;
    qudt:iec61360Code "0112/2///62720#UAA597" ;
    ...
.
```

but I don't need all of those for the vocab.

I've created a slimmed down QK vocab for you (will attach it after this) and you can see that vocab displayed in my test GSQ test VocPrez at http://gsq.surroundaustralia.com/object?uri=https://linked.data.gov.au/def/qesd-qkd

* Got to https://shacl.org/playground/ and put this file https://w3id.org/profile/vocpub/validator in the "Shapes Graph" text area and the vocab in the "Data Graph" text area, set the formats to Turtle and hit validate

**Comment by broughd-qld** [comment]
Thanks @nicholascar. I was just going for a copy and paste from QUDT - but your suggestions make it easier to manage.

**Comment by broughd-qld** [comment]
@nicholascar @KellyVance  - Updated as requested and re-validated with no errors in shacl.org/playground.

@nicholascar  - I have put back in the links between Quantity Kinds (qudt:hasQuantityKind) and Applicable Units (qudt:applicableUnit). The intention is that we will include Quantity Kinds or Units in our observed properties vocabulary to limit the list of units that a user can select.

**Review by nicholascar — APPROVED** [review]
Looks good!

**Review by KellyVance — APPROVED** [review]

**Review by UbuntuScott — APPROVED** [review]
Seems to have everything we need for WQ at a cursory glance as well @broughd-qld

**Pull request merged**